utils/db_utils.py

```diff
@@ -106,10 +106,13 @@ def query_demographics():
     if not df.empty:
         for col in constants.BINARY_DEMOGRAPHICS_COLS:
             if col in df.columns:
-                df[col] = df[col].apply(str)  
-    df.drop(columns=['data.xmlResponse', 'data.name', 'data.version', 'data.label'], inplace=True) 
+                df[col] = df[col].apply(str) 
+    columns_to_drop = [col for col in df.columns if col.startswith("metadata")]
+    df.drop(columns= columns_to_drop, inplace=True) 
+    df.drop(columns=['data.xmlResponse', 'data.name', 'data.version', 'data.label', 'data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.attr.id','data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.attr.xmlns:jr','data.jsonDocResponse.aSfdnWs9LE6q8YEF7u9n85.attr.xmlns:orx'], inplace=True) 
```

Comment:
I don't think we should hardcode any of the form ids (such as `aSfdnWs9LE6q8YEF7u9n85`) because I believe they can be different from form to form. But the other changes are fine.

Reply:
e.g. the form ID for the time use survey, for example, is `time_use_survey_form_v9_1`
https://github.com/sebastianbarry/nrel-openpath-deploy-configs/blob/surveys-info-and-surveys-data/survey-resources/data-json/time-use-survey-form-v9.json